### ISSUE TITLE
Is there any good way to apply translations xml file to original json?

### ISSUE THREAD

**Issue by shalomeir** [emmenko/redux-react-router-async-example#46]
I'd like to apply all '_translations' file that I translated to apply to original json such as 'lib/i18n/de.js'. Is there any good way to apply translations xml file to original json? I couldn't find automatic ways. Do I have to copy and paste per each changed sentence.

**Comment by emmenko** [comment]
Yeah, I haven't got time to implement this, but I wanted to have a script (similar to [this](https://github.com/emmenko/redux-react-router-async-example/blob/master/scripts/i18nToXliff.js)) that parses the xml file and output the i18n javascript file.

I hope to provide it soon (I also need it in my company), in the meantime manual copy / paste ;)

**Comment by shalomeir** [comment]
@emmenko I got it. This example is very helpful for our project. I'm very appreciated for sharing this program. Also I'll wait for your implementation about this. Thanks!
